docs/developers/MicroBenchmarks.md

```diff
@@ -94,23 +94,14 @@ InputFromBatchVector/iterations:1/process_time/real_time/threads:1   41304520 ns
 
 ## Generate Substrait plan and input for any query
 
-First, rebuild Gluten with build type `RelWithDebInfo`.
-
-```shell
-cd /path_to_gluten/
-./dev/buildbundle-veloxbe.sh --build_tests=ON --build_benchmarks=ON --build_type=RelWithDebInfo
-
-```
-Run the query by spark-shell.  Get the Substrait plan from executor's stdout.
+Run the query with config `spark.gluten.sql.debug=true` and get the Substrait json plan from log, then save it as a JSON file, suppose the name is "plan.json".
```

Comment:
We also need config `spark.gluten.sql.columnar.backend.velox.glogSeverityLevel=0`

Reply:
thank you, btw could we open glog when debug mode enabled?

Reply:
> thank you, btw could we open glog when debug mode enabled?

Sounds good!